app/containers/App/sagas.js

```diff
@@ -119,6 +134,155 @@ export function* validateTokenSaga() {
   }
 }
 
+export function* saveEntitySaga({ data }) {
+  try {
+    yield put(saveSending());
+
+    // update entity attributes
+    // on the server
+    const entityUpdated = yield call(updateEntityRequest, data.path, data.entity);
```

Comment:
uses entity-update helper functions to get api requests (https://github.com/dumparkltd/hr-nmrf-client/pull/92/files#diff-ebe7fe3af1b5b6012c218f889d8f4508)